services/worker/src/worker/job_runners/config/parquet_metadata.py

```diff
@@ -26,17 +28,34 @@
     ParquetFileMetadataItem,
 )
 from worker.job_runners.config.config_job_runner import ConfigJobRunner
-from worker.utils import get_parquet_file
+from worker.utils import hf_hub_open_file, hffs_parquet_url
 
 SLEEPS = [0.2, 1, 1, 10, 10, 10]
 
 
+# TODO: on which error?
+@retry(sleeps=SLEEPS)
```

Comment:
not sure which error I should retry on, in `config-parquet-and-info`, which basically does the same thing, it's `pa.ArrowInvalid`: 
https://github.com/huggingface/dataset-viewer/blob/e7e674ad8dcd0f8fcd1e2ede0591fd8c0afb4ee3/services/worker/src/worker/job_runners/config/parquet_and_info.py#L595C26-L595C38
But I'm not sure where this error might come from.

Reply:
Not sure either, I had this error from time to time in prod.

I added this retry in https://github.com/huggingface/dataset-viewer/pull/1404

Maybe corrupted queries ?

Reply:
I guess you can use pa.ArrowInvalid as well